task/verify-enterprise-contract/0.1/verify-enterprise-contract.yaml

```diff
@@ -103,6 +107,15 @@ spec:
         value: "$(params.HOMEDIR)"
 
   steps:
+    - name: annotate-task
```

Comment:
Why is this required? The annotations from the definition are present on the TaskRun. We do not want to allow TaskRuns to modify their own metadata this could lead to all sort of security issues.

Reply:
PAC is removing these custom annotations which is required for RHDH UI to identify the ec task to show the reports, so this seems like a workaround to overcome this issue. Similar fix is done in ACS and SBOM tasks ref- https://github.com/redhat-appstudio/build-definitions/pull/859 . Tagging @jduimovich who can add more details to this.